### PR TITLE
added before after toggle button

### PR DESCRIPTION
**Pull request by vinaysajjanapu**
Fixes issue #584 

Changes: - Added before after button
                - created a copy of original bitmap and shown that when button is touched.

### CODE REVIEW

File: app/src/main/java/vn/mbm/phimp/me/editor/editimage/EditImageActivity.java [hunk 1]
```diff
@@ -78,14 +79,15 @@
 
     private EditImageActivity mContext;
     public Bitmap mainBitmap;
+    private Bitmap originalBitmap;
     public ImageViewTouch mainImage;
-    private View cancel,save;
+    private View cancel,save,bef_aft;
```
Comment (vinaysajjanapu) [inline app/src/main/java/vn/mbm/phimp/me/editor/editimage/EditImageActivity.java]: @pa1pal @anantprsd5 @mohitmanuja @Subhankar29 
Is this change necessary?

Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/editor/editimage/EditImageActivity.java]: @vinaysajjanapu not mandatory. It is pointing because the variable `bef_aft` only used in the scope of `onCreate` block. 
So can be replaced by local variable

**Comment by mohitmanuja** [comment]
@vinaysajjanapu Please resolve the conflicts.

**Comment by vinaysajjanapu** [comment]
@mohitmanuja  done
